local-testing-docs/environment.yml

```diff
@@ -0,0 +1,12 @@
+# Snowpark environment file
+# https://docs.snowflake.com/en/developer-guide/snowpark/python/setup.html
+
+name: snowpark-local-test
```

Comment:
What is this file used for?

Reply:
Should we keep these files (in `local-testing-docs/`) here or moved to a separate repo?

Reply:
good point, to control the scope of the PR, I will remove it now